packages/core/scripts/serverless-orchestration/ServerlessHub.js

```diff
@@ -60,7 +60,9 @@ hub.post("/", async (req, res) => {
     // Fetch the last block number this given config file queried the blockchain at if running in production. Else, pull from env.
     const lastQueriedBlockNumber = await _getLastQueriedBlockNumber(req.body.configFile);
     if (!configObject || !lastQueriedBlockNumber)
-      throw new Error("Serverless hub requires a config object and a last updated block number!");
+      throw new Error(
+        `Serverless hub requires a config object and a last updated block number! configObject:${configObject} lastQueriedBlockNumber:${lastQueriedBlockNumber}`
```

Comment:
Does configObject need to be stringified?

Reply:
sure & done.